code/game/gamemodes/wizard/artifacts.dm

```diff
@@ -29,6 +29,13 @@
 	price = 2 * Sp_BASE_PRICE
 	spawned_items = list(/obj/item/weapon/gun/energy/staff/change)
 
+/datum/spellbook_artifact/staff_of_swapping
+	name = "Staff of Swapping"
+	desc = "An artefact that fires a glowing bolt of energy which transfers the caster and targets position in space. Wielding in it both hands increases the power of the staff, and allows it to pass through certain objects.."
```

Comment:
Should be **artifact** instead of _artefact_, right?

Reply:
Yep, but all the other artifacts were spelt "artefact" as well, so I just kind of assumed it was that way for a reason.

Reply:
Oh nvm, I love differences in spelling based on country!!!

Reply:
It's magyk, I ain't gotta explain shit